cns/service/main.go

```diff
@@ -1039,6 +1041,18 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 		return errors.Wrapf(err, "failed to setup reconciler with manager")
 	}
 
+	// adding some routes to the root service mux
+	mux := httpRestServiceImplementation.Listener.GetMux()
+	mux.Handle("/readyz", http.StripPrefix("/readyz", &healthz.Handler{}))
+	if cnsconfig.EnablePprof {
```

Comment:
pprof is a pretty effective DOS so I do not want it exposed accidentally - this listener is on the hostnet which means it could end up publicly accessible

Reply:
In that case maybe we should consider having pprof on localhost? If the default is for these to not be enabled, then it seems like collecting meaningful profiles could be more difficult, since you will need a process restart and wait for conditions/leaks to reproduce. We would also need changes in the CNS-SF wrapper project for providing this functionality to other orchestration teams.